pbfhr/gFHR/steady/gFHR_griffin_cr_ss.i

```diff
@@ -1,534 +1,21 @@
-  []
-  [Burnup_avg]
-    type = PebbleAveragedAux
-    block = 'pebble_bed'
-    variable = Burnup_avg
-    array_variable = Burnup
-    pebble_volume_fraction = pebble_volume_fraction
-    n_fresh_pebble_types = 1
-    execute_on = 'INITIAL TIMESTEP_END'
-  []
-  [porosity]
-    type = FunctionAux
-    block = 'pebble_bed'
-    variable = porosity
-    function = porosity_f
-    execute_on = 'INITIAL'
-  []
-[]
-
-[Functions]
-  [porosity_f]
-    type = PiecewiseMulticonstant
-    direction = 'right right'
-    data_file = '../data/gFHR_porosity.txt'
-  []
-  [CR_pos_f]
-    type = ConstantFunction
-    value = '3.38523' #Note that when position is 3.6947 CR is withdrawn
-  []
-[]
-# ==============================================================================
-# DEPLETION
-# ==============================================================================
-[PebbleDepletion]
-  block = 'pebble_bed'
-
-  power = ${total_power}
-  integrated_power_postprocessor = total_power
-  power_density_variable = total_power_density
-  family = MONOMIAL
-  order = CONSTANT
-
-  burnup_grid_name = 'Burnup'
-  fuel_temperature_grid_name = 'Tfuel'
-  moderator_temperature_grid_name = 'Tmod'
-  additional_grid_name_variable_mapping = 'Rho Rho'
-
-  # transmutation data
-  dataset = ISOXML
-  isoxml_data_file = '../data/DRAGON5_DT.xml'
-  isoxml_lib_name = 'DRAGON'
-  dtl_physicality = DISABLE
-
-  # Isotopic options
-  n_fresh_pebble_types = 1
-  fresh_pebble_compositions = pebble
-  track_isotopes = 'U235 U236 U238 PU238 PU239 PU240 PU241 PU242 AM241 AM242M CS135 CS137 XE135 XE136 I131 I135 SR90'
-  decay_heat = true
-
-  # Pebble options
-  porosity_name = porosity
-  burnup_group_boundaries = ${burnup_group_boundaries}
-
-  # Tabulation data.
-  initial_moderator_temperature = '${initial_temperature}'
-  initial_fuel_temperature = '${initial_temperature}'
-
-  # coolant settings
-  coolant_composition_name = coolant
-  coolant_density_variable = 'Rho'
-  coolant_density_ref = ${Rho_ref}
-  # TODO: add material id 2 with coolant as well
-  coolant_material_id = '1'
-
-  [DepletionScheme]
-    type = ConstantStreamlineEquilibrium
-
-    # Streamline definition
-    major_streamline_axis = y
-    streamline_points = '0.152  0.60 0  0.152  3.6947 0;
-                         0.450  0.60 0  0.450  3.6947 0;
-                         0.750  0.60 0  0.750  3.6947 0;
-                         1.050  0.60 0  1.050  3.6947 0'
-    streamline_segment_subdivisions = '20; 20; 20; 20' # align with the neutronics mesh
-
-    # Pebble options
-    pebble_unloading_rate = ${pebble_unloading_rate}
-    pebble_flow_rate_distribution = ${streamline_flow_fraction}
-    pebble_diameter = '${fparse pebble_radius * 2.0}'
-    burnup_limit = '${burnup_limit}'
-
-    # Solver parameters
-    sweep_tol = 1e-8
-    sweep_max_iterations = 100
-
-    # Output
-    exodus_streamline_output = false
-  []
-
-  # pebble conduction
-  pebble_conduction_input_file = 'gFHR_pebble_triso_ss.i'
```

Comment:
remind me again why you cant just swap this input to use Griffin instead of BlueCrab?

what else in the input is BlueCrab?

Reply:
Because it uses materials of type `PronghornSteadyStateSolidMaterial` and UserObjects that depend on these materials.

Reply:
right but if you passed  pebble_conduction_input_file = 'gFHR_pebble_triso_neutronics_only.i' you would not see any of those?

Reply:
Oh, I think I see what you're saying. Instead of using the `testing/gFHR_griffin_cr_ss_neutronics_only.i` input, you want to use `gFHR_griffin_cr_ss.i` and pass the `pebble_conduction_input_file` and `flow_subapp_input_file` as cli_args?

Reply:
I think that would work.
And you can avoid the multiple inputs in the neutronics input